api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -170,10 +171,9 @@ def aspirate(
                          robot will aspirate from the exact specified location.
                          If unspecified, the robot will aspirate from the
                          current position.
-        :param rate: A relative modifier for how quickly to aspirate liquid.
-                     The flow rate for this aspirate will be
-                     `rate` * :py:attr:`flow_rate.aspirate <flow_rate>`.
-                     If not specified, defaults to 1.0.
+        :param rate: How quickly a pipette aspirates liquid, measured in µL/s.
```

Comment:
This parameter is *not* measured in µL/s. 200 sets it to 200x the default rate, not 200 µL/s. 

The old copy is a little clunky though.
```suggestion
        :param rate: A multiplier for the default flow rate of the pipette.
```